core_api/src/semantic_routes.py

```diff
@@ -101,7 +106,30 @@
 )
 
 
-routes = [info, ability, coach, gratitude, summarisation, extract]
+def get_semantic_routes():
+    return [info, ability, coach, gratitude, summarisation]
+
+
+def get_semantic_routing_encoder():
+    return HuggingFaceEncoder(name="sentence-transformers/paraphrase-albert-small-v2", cache_dir=MODEL_PATH)
```

Comment:
note to self, this should come from Settings.emebedding_model (not urgent though)